eng/common/README.md

```diff
@@ -10,14 +10,3 @@ Any updates to files in the `eng/common` directory should be made in the [azure-
 All changes made will cause a PR to created in all subscribed azure-sdk language repos which will blindly replace all contents of
 the `eng/common` directory in that repo. For that reason do **NOT** make changes to files in this directory in the individual azure-sdk
 languages repos as they will be overwritten the next time an update is taken from the common azure-sdk-tools repo.
```

Comment:
@chidozieononiwu  this doesn't look correct. Can you try to understand why it looks like it is removing these things you recently added?

Reply:
I suspect it might have to do with the fact that @sima-zhu hasn't merged with your latest tools changes. We need to figure out a good way to make sure we don't accidently undo changes in this way.

Reply:
It seems that @sima-zhu changes occurred somewhere while I was working on it. Please @sima-zhu can you rebase the Tools PR and push the changes, that should fix all the Sync PRs.

Reply:
I have merged the conflicts. It disappeared now.
@chidozieononiwu Could you check the diff? I will merge the change once you approved.